web/src/views/Settings/Security/SecurityView.tsx

```diff
@@ -0,0 +1,244 @@
+    };
+
+    return (
+        <Fragment>
+            <SecondFactorDialog
+                info={userInfo}
+                elevation={elevation}
+                opening={dialogSFOpening}
+                handleClosed={handleSFDialogClosed}
+                handleOpened={handleSFDialogOpened}
+            />
+            <IdentityVerificationDialog
+                opening={dialogIVOpening}
+                elevation={elevation}
+                handleClosed={handleIVDialogClosed}
+                handleOpened={handleIVDialogOpened}
+            />
+            <ChangePasswordDialog
+                username={userInfo?.display_name || ""}
+                open={dialogPWChangeOpen}
+                setClosed={() => {
+                    handleResetState();
+                }}
+            />
+
+            <Container
+                sx={{
+                    display: "flex",
+                    justifyContent: "center",
+                    alignItems: "flex-start",
+                    height: "100vh",
+                    pt: 8,
+                }}
+            >
+                <Paper
+                    variant="outlined"
+                    sx={{
+                        display: "flex",
+                        justifyContent: "center",
+                        alignItems: "center",
+                        height: "auto",
+                    }}
+                >
+                    <Stack spacing={2} sx={{ m: 2, width: "100%" }}>
+                        <Box sx={{ p: { xs: 1, md: 3 } }}>
+                            <Box
+                                sx={{
+                                    width: "100%",
+                                    p: 1.25,
+                                    mb: 1,
+                                    border: `1px solid ${theme.palette.grey[600]}`,
+                                    borderRadius: 1,
+                                }}
+                            >
+                                <Box display="flex" alignItems="center">
+                                    <Typography sx={{ mr: 1 }}>{translate("Email")}:</Typography>
+                                    <Typography>{userInfo?.emails?.[0] || ""}</Typography>
+                                </Box>
+                                {userInfo?.emails && userInfo.emails.length > 1 && (
+                                    <List sx={{ width: "100%", padding: 0, pl: 4 }}>
+                                        {" "}
+                                        {userInfo.emails.slice(1).map((email: string, index: number) => (
+                                            <ListItem key={index} sx={{ paddingTop: 0, paddingBottom: 0 }}>
+                                                <Typography>{email}</Typography>
+                                            </ListItem>
+                                        ))}
+                                    </List>
+                                )}
+                            </Box>
```

Comment:
I'm curious about the intent on adding this now. Is it just displaying information?